samples/maps/demo/data-class-two-ranges/demo.css

```diff
@@ -10,3 +10,40 @@
     text-align: center;
     color: gray;
 }
+
+#popup-pie {
+    width: 290px;
+    height: 240px;
+    padding: 0.5em 1em;
+    cursor: default;
```

Comment:
`cursor: default` is ... default so the declaration is unnecessary.

Reply:
not in this context, as this is draggable annotation, so default here is drag pointer, not default cursor, and for better UX it seems better to stick to default pointer